etc/gatein/portal/portal/picketlink-idm-MSAD-config.xml.template

```diff
@@ -0,0 +1,422 @@
+                <type>text</type>
+                <isRequired>false</isRequired>
+                <isMultivalued>false</isMultivalued>
+                <isReadOnly>false</isReadOnly>
+              </attribute>
+            </attributes>
+            <options>
+              <option>
+                <name>idAttributeName</name>
+                <value>cn</value>
+              </option>
+              <option>
+                <name>ctxDNs</name>
+                <value>ou=Organization,o=gatein,${USER_DIRECTORY_BASE_DN}</value>
+              </option>
+              <option>
+                <name>entrySearchFilter</name>
+                <value><![CDATA[(&(cn={0})(objectClass=group))]]></value>
+              </option>
+              <option>
+                <name>allowCreateEntry</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>parentMembershipAttributeName</name>
+                <value>member</value>
+              </option>
+              <option>
+                <name>isParentMembershipAttributeDN</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>allowEmptyMemberships</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>createEntryAttributeValues</name>
+                <value>objectClass=top</value>
+                <value>objectClass=group</value>
+                <value>groupType=8</value>
+              </option>
+            </options>
+          </identity-object-type>
+        </supported-identity-object-types>
+        <options>
+          <option>
+            <name>providerURL</name>
+            <value>ldap://localhost:${DEPLOYMENT_LDAP_PORT}</value>
+          </option>
+          <option>
+            <name>adminDN</name>
+            <value>${USER_DIRECTORY_ADMIN_DN}</value>
+          </option>
+          <option>
+            <name>adminPassword</name>
+            <value>${USER_DIRECTORY_ADMIN_PASSWORD}</value>
+          </option>
+          <option>
+            <name>authenticationMethod</name>
+            <value>simple</value>
+          </option>
+          <option>
+            <name>customSystemProperties</name>
+            <value>javax.net.ssl.trustStore=/path/to/msad.truststore</value>
+            <value>javax.net.ssl.trustStorePassword=password</value>
+          </option>
+          <option>
+            <name>searchTimeLimit</name>
+            <value>10000</value>
+          </option>
+          <option>
+            <name>createMissingContexts</name>
+            <value>true</value>
+          </option>
+          <option>
+            <name>customJNDIConnectionParameters</name>
+            <value>com.sun.jndi.ldap.connect.pool=true</value>
+          </option>
+          <option>
+            <name>customSystemProperties</name>
+            <value>com.sun.jndi.ldap.connect.pool.maxsize=300000</value>
```

Comment:
the same here : 
This value should not be kept as default, it's way too much, there is a confusion between timeout and pool size cf https://jira.exoplatform.org/browse/EXOGTN-2398